website/blog/new-graphback-release-0-14.md

```diff
@@ -0,0 +1,158 @@
+---
+title: New release of Graphback - GraphQL Data Access methods for your project
+author: Manyanda Chitimbo
+author_title: Software Engineer
+author_url: https://github.com/machi1990
+author_image_url: https://avatars3.githubusercontent.com/u/10106536?s=460&u=ba1f66c0295c5151f147dd869a543bdc68671c56&v=4
+tags: [graphql, graphback, graphqlcrud, graphql-js, nodejs, release]
+image: https://graphback.dev/img/logo.png
+---
+
+# TL;DR:
+Over the past months, we have been working on improving Graphback. We are happy to announce that we have released Grapback 0.14.0. 
+Most notable changes in this release are:
+
+- Full implementation of the CRUD API specification
+- Improved runtime API to remove complexity of setting up a Graphback API
+- RBAC Authorization on top of the Graphback Service and Keycloak
+- Improved documentation and website
```

Comment:
I would omit this, the docs go hand in hand with the feature changes and should be a given.